ledfx/core.py

```
@@ -233,10 +234,28 @@
 
             time_since_last[vis_id] = time_now
 
+            rows = getattr(event, "rows")
```

Ruff

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)